<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>600</width><height>580</height></rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>

      <!-- Page 0: Login/Role Selection -->
      <widget class="QWidget" name="page_login">
       <layout class="QVBoxLayout" name="verticalLayout">

        <item>
         <spacer name="topSpacer">
          <property name="orientation"><enum>Qt::Vertical</enum></property>
          <property name="sizeHint"><size><width>20</width><height>30</height></size></property>
         </spacer>
        </item>

        <item>
         <widget class="QLabel" name="welcomeLabel">
          <property name="text">
           <string>Welcome to our Inventory System</string>
          </property>
          <property name="alignment"><set>Qt::AlignCenter</set></property>
         </widget>
        </item>

        <item>
         <widget class="QLabel" name="roleLabel">
          <property name="text">
           <string>Please select your role</string>
          </property>
          <property name="alignment"><set>Qt::AlignCenter</set></property>
         </widget>
        </item>

        <item>
         <widget class="QPushButton" name="adminBtn">
          <property name="text"><string>Admin</string></property>
         </widget>
        </item>

        <item>
         <widget class="QPushButton" name="userBtn">
          <property name="text"><string>User</string></property>
         </widget>
        </item>

        <item>
         <widget class="QGroupBox" name="loginGroup">
          <property name="title"><string>Login / Register</string></property>
          <layout class="QVBoxLayout" name="loginLayout">

           <item>
            <widget class="QLineEdit" name="idInput">
             <property name="placeholderText"><string>Enter ID</string></property>
            </widget>
           </item>

           <item>
            <widget class="QLineEdit" name="passwordInput">
             <property name="echoMode"><enum>QLineEdit::Password</enum></property>
             <property name="placeholderText"><string>Enter Password</string></property>
            </widget>
           </item>

           <item>
            <widget class="QCheckBox" name="showPasswordCheckbox">
             <property name="text"><string>Show Password</string></property>
            </widget>
           </item>

           <item>
            <layout class="QHBoxLayout" name="loginButtonsLayout">
             <item>
              <widget class="QPushButton" name="loginBtn">
               <property name="text"><string>Login</string></property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="registerBtn">
               <property name="text"><string>Register</string></property>
              </widget>
             </item>
            </layout>
           </item>

           <item>
            <widget class="QLabel" name="messageLabel">
             <property name="text"/>
             <property name="alignment"><set>Qt::AlignCenter</set></property>
            </widget>
           </item>

           <item>
            <widget class="QPushButton" name="backBtn">
             <property name="text"><string>Back</string></property>
            </widget>
           </item>

          </layout>
         </widget>
        </item>

        <item>
         <spacer name="bottomSpacer">
          <property name="orientation"><enum>Qt::Vertical</enum></property>
          <property name="sizeHint"><size><width>20</width><height>40</height></size></property>
         </spacer>
        </item>

       </layout>
      </widget>

      <!-- Page 1: User Page container -->
      <widget class="QWidget" name="page_user">
       <layout class="QVBoxLayout" name="userLayout">
        <item>
         <widget class="QWidget" name="userPageContainer">
          <property name="minimumSize">
           <size><width>400</width><height>400</height></size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Page 2: Checkout Page -->
      <widget class="QWidget" name="checkoutPage">
       <layout class="QVBoxLayout" name="checkoutLayout">
        <item>
         <widget class="QLabel" name="checkoutLabel">
          <property name="text">
           <string>Checkout Page</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
